src/hash.cpp

```diff
@@ -24,9 +24,23 @@ namespace mococrw
 {
 using namespace openssl;
 
+static const std::string SHA1 = "sha1";
```

Comment:
Wasn't the story about sha384 only?

Reply:
After external confirmation we ended up exposing SHA1 as well.